src/runtime/contrib/tensorrt/tensorrt_runtime.cc

```diff
@@ -125,17 +147,26 @@ class TensorRTRuntime : public JSONRuntimeBase {
 
   /*! \brief Run inference using built engine. */
   void Run() override {
+    
```

Comment:
Suggest to create calibrator and process the input information here, if there is still some remaining calibration batches, just return.

This can get rid of building an engine then destory it and build one again.